spec/javascripts/admin/modules/unpublish-display-conditions.spec.js

```diff
@@ -95,4 +106,21 @@ describe('GOVUK.Modules.UnpublishDisplayConditions', function () {
     expect(body.querySelector('.js-unpublish-withdraw-form__consolidated').style.display).toEqual('block')
     expect(body.querySelector('.js-unpublish-withdraw-form__withdrawal').style.display).not.toEqual('block')
   })
+
+  it('should hide Public explanation section when Redirect to URL automatically is pre-checked', function () {
+    expect(body.querySelector('.js-unpublish-withdraw-form__published-in-error div.gem-c-textarea').style.display).not.toEqual('block')
```

Comment:
minor, seems like we can assert on something explicit. Knowing it's not block doesn't mean it's actually hidden.

```suggestion
    expect(body.querySelector('.js-unpublish-withdraw-form__published-in-error div.gem-c-textarea').style.display).toEqual('hidden')
```